src/Illuminate/Support/Testing/Fakes/BusFake.php

```diff
@@ -187,10 +187,10 @@ public function assertNotDispatchedAfterResponse($command, $callback = null)
     /**
      * Assert if a batch was dispatched based on a truth-test callback.
      *
-     * @param  callable  $callback
+     * @param  callable|null  $callback
      * @return void
      */
-    public function assertBatched(callable $callback)
+    public function assertBatched(callable $callback = null)
```

Comment:
These are breaking changes to the method signatures.

Reply:
PHP seems to be not complaining about it:

```php
>>> class A { public function foo(callable $callback) {} }
>>> class B extends A { public function foo(callable $callback = null) {} }
```

Reply:
Oh sorry, that's not our case.

This is ours:

```php
>>> class A { public function foo(callable $callback = null) {} }
>>> class B extends A { public function foo(callable $callback) {} }
PHP Warning:  Declaration of B::foo(callable $callback) should be compatible with A::foo(?callable $callback = NULL) in Psy Shell code on line 3
```

Yeah, it throws a warning.